spec/lib/guard/puma_spec.rb

```diff
@@ -12,6 +12,18 @@
 
       expect(guard.runner.options[:port]).to eq(4000)
     end
+
+    context "when :config is set" do
```

Comment:
`context "when config option is set" do`

Reply:
Done.